src/integration-test/tests/citizen/endToEnd/part_admission_test.ts

```diff
@@ -0,0 +1,377 @@
+const claimantDashboardPage: ClaimantDashboardPage = new ClaimantDashboardPage()
+const claimantTaskListPage: ClaimantTaskListPage = new ClaimantTaskListPage()
+const claimantDefendantResponsePage: ClaimantDefendantResponsePage = new ClaimantDefendantResponsePage()
+const claimantSettleAdmittedPage: ClaimantSettleAdmittedPage = new ClaimantSettleAdmittedPage()
+const claimantCheckAndSendPage: ClaimantCheckAndSendPage = new ClaimantCheckAndSendPage()
+const claimantConfirmation: ClaimantConfirmation = new ClaimantConfirmation()
+const directionsQuestionnaireSteps: DirectionsQuestionnaireSteps = new DirectionsQuestionnaireSteps()
+const defendantDashboardPage: DefendantDashboardPage = new DefendantDashboardPage()
+const claimantAcceptPaymentMethod: ClaimantAcceptPaymentMethod = new ClaimantAcceptPaymentMethod()
+const claimantChooseHowToProceed: ClaimantChooseHowToProceed = new ClaimantChooseHowToProceed()
+const claimantSignSettlementAgreement: ClaimantSignSettlementAgreement = new ClaimantSignSettlementAgreement()
+const defendantSignSettlementAgreement: DefendantSignSettlementAgreement = new DefendantSignSettlementAgreement()
+const defendantSignSettlementAgreementConfirmation: DefendantSignSettlementAgreementConfirmation = new DefendantSignSettlementAgreementConfirmation()
+const defendantPaidAnyMoneyPage: DefendantPaidAnyMoneyPage = new DefendantPaidAnyMoneyPage()
+const paidAmountSummaryPage: PaidAmountSummaryPage = new PaidAmountSummaryPage()
+const claimantPaymentOptionPage: ClaimantPaymentOptionPage = new ClaimantPaymentOptionPage()
+const claimantCourtOfferedInstalmentsPage: ClaimantCourtOfferedInstalmentsPage = new ClaimantCourtOfferedInstalmentsPage()
+const defendantSettlementAgreementRepaymentPlanSummary: DefendantSettlementAgreementRepaymentPlanSummary = new DefendantSettlementAgreementRepaymentPlanSummary()
+
+async function prepareClaim (I: I) {
+  const claimantEmail: string = userSteps.getClaimantEmail()
+  const defendantEmail: string = userSteps.getDefendantEmail()
+
+  const claimData: ClaimData = createClaimData(PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+  const claimRef: string = await I.createClaim(claimData, claimantEmail)
+
+  await helperSteps.enterPinNumber(claimRef, claimantEmail)
+  helperSteps.linkClaimToDefendant(defendantEmail)
+  helperSteps.startResponseFromDashboard(claimRef)
+
+  return { data: claimData, claimRef: claimRef }
+}
+
+if (process.env.FEATURE_ADMISSIONS === 'true') {
+  Feature('Partially admit the claim')
+
+  Scenario('I can complete the journey when I partially admit the claim with payment already made @citizen @admissions', { retries: 3 }, async (I: I) => {
+    const claimData = await prepareClaim(I)
+    defenceSteps.makePartialAdmission(claimData.data.defendants[0])
+    defenceSteps.partialPaymentMade(PartyType.INDIVIDUAL)
+  })
+
+  Scenario('I can complete the journey when I partially admit the claim with immediate payment @nightly @admissions', { retries: 3 }, async (I: I) => {
```

Comment:
Are these three tests not duplicates of the ones below? If not can you refactor the tests below to use these so that each one has a different payment option and then remove these three?

Reply:
There three are basic test to check part admit journey is working and it is going to be @citizen tag and remaining I am going to change to @nightly with more steps involved after claim as submitted as part admit with immediate, set date and instalments